gym/spaces/discrete.py

```diff
@@ -10,6 +10,9 @@ class Discrete(gym.Space):
     def __init__(self, n):
         self.n = n
     def sample(self):
+        # prng is terminating VizDoom with Fatal Error - Address not mapped to object (signal 11)
+        # 82 is for HexEnv (9x9), which is the environment causing the Vizdoom crash
+        if 82 == self.n: return int(round(time.time(), -1) % self.n)
```

Comment:
Re: the test crash, I'd like to find a more robust fix. If it's just DoomDeathmatch that's affected, can we temporarily disable it / re-enable it afterwards?

Ready to merge this PR after that.